zerver/models.py

```diff
@@ -1446,6 +1446,12 @@ class PushDeviceToken(AbstractPushDeviceToken):
 
     # The user whose device this is
     user: UserProfile = models.ForeignKey(UserProfile, db_index=True, on_delete=CASCADE)
+    # Encryption
+    #
+    # AESGCM.generate_key() generates a 16 byte long key which is converted to
+    # a url-safe base64 encoded string. 16 bytes can be represented by 24 base64
+    # characters, so max_length of 25 should be safe.
+    notification_encryption_key: Optional[str] = models.CharField(max_length=25, null=True)
```

Comment:
I think we might be better off using a `TextField` so that we have flexibility if we later change the key algorithm.